vanilla-javascript-demo/README.md

```diff
@@ -0,0 +1,68 @@
+# Using ChemDraw JS with Vanilla JavaScript
+
+Example of how to use the ChemDraw JS library with vanilla JavaScript and HTML5.
+
+This example demonstrates how to load the ChemDraw JS library when installed as an npm package. It is based on the example in the __ChemDraw JS Developer's Guide__. For a project without npm the same code can be used, just change the URL to the library as appropriate.
+
+For this demo, copy the license into /dependencies.
```

Comment:
Isn't this is a repeat of the text below? The instructions now seem to be in the wrong order.

Read this from the top, imagining that you're a new user looking at CDJS for the first time. If you follow the instructions here it won't work because you've not put CDJS in the right place.

Reply:
> Isn't this is a repeat of the text below? The instructions now seem to be in the wrong order.
> 
> Read this from the top, imagining that you're a new user looking at CDJS for the first time. If you follow the instructions here it won't work because you've not put CDJS in the right place.

Step orders adjusted